# Guide Tree Optimization with Genetic Algorithm to Improve Multiple Alignment of Protein 3D-structures

With the increasing availability of 3D-data, the focus of bioinformatic analysis
is shifting from protein sequence alignments towards more content-rich 3D-alignments.
This raises the need for new ways to improve the accuracy of 3D-superimposition.

We propose guide tree optimization with genetic algorithm (GA) as a universal tool
to improve the alignment quality of multiple protein 3D-structures systematically.
As a proof of concept, the suggested GA-based approach was implemented in popular
[Matt](http://matt.cs.tufts.edu/) and [Caretta](http://www.bioinformatics.nl/caretta/)
multiple protein 3D-structure alignment algorithms leading to a statistically significant
improvement of the [TM-score](https://zhanglab.ccmb.med.umich.edu/TM-score/) quality indicator by up to 220-1523\%
on "SABmark Superfamilies" (in 49-77% of cases) and "SABmark Twilight" (in 59-80% of cases)
datasets.
The observed improvement of TM-score in collections of distant homologues highlights
the potentials of GA to optimize 3D-alignments of diverse protein superfamilies
therefore facilitating the study of structure-function relationship.

[gaMatt](https://github.com/n-canter/gamaps/tree/main/gaMatt) is a GA version of Matt and
[gaCaretta](https://github.com/n-canter/gamaps/tree/main/gaCaretta) is a GA version of Caretta.

# Contacts and support
* Maksim Shegay: `max.shegai [at] gmail.com`
* Dmitry Suplatov: `d.a.suplatov [at] belozersky.msu.ru`

# Citing
M. Shegay, V. Å vedas, Vl. Voevodin, D. Suplatov, N. Popova (2021). Guide Tree Optimization with Genetic Algorithm to Improve Multiple Alignment of Protein 3D-structures.


